client/rpc/rpcclient_test.go

```diff
@@ -0,0 +1,123 @@
+package rpcclient
+
+import (
+	"testing"
+
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/rpcclient"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
```

Comment:
```suggestion
var _ client.Client = (*RPCClient)(nil)
```

When creating implementations of interfaces it is nice to have this kind of interface compliance verification to ensure RPCClient is an implementation of the Client interface at compile time.

Reply:
Uber has a nice [guide](https://github.com/uber-go/guide/blob/master/style.md#verify-interface-compliance) on this kind of stuff.

Reply:
Oh that's cool, makes sense given interfaces are implemented implicitly in Go.